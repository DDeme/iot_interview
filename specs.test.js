const evaluateLogFile = require('./src/evaluateLogFile')
const fs = require('fs')
const fileString = fs.readFileSync('./input/data.txt', 'utf-8')

describe('test of simple use case ', () => {
  test('1st sample', () => {
    expect(evaluateLogFile(fileString)).toMatchSnapshot()
  })
})
