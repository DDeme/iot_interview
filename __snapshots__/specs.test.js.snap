// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test of simple use case  1st sample 1`] = `
Object {
  "hum-1": "keep",
  "hum-2": "discard",
  "temp-1": "presise",
  "temp-2": "ultra presise",
}
`;
